platform-emf/src/main/java/com/softicar/platform/emf/data/table/export/conversion/TableExportTypedNodeConverter.java

```diff
@@ -285,7 +285,12 @@ else if (node instanceof ITableExportTypedNode) {
 
 		else if (node instanceof DomTextArea) {
 			debugPrint(currentDepth, "DomTextArea");
-			values.add(createStringExportNodeValue(((DomTextArea) node).getInputTextTrimmed()));
+			values.add(createStringExportNodeValue(((DomTextArea) node).getValueTextTrimmed()));
+		}
+
+		else if (node instanceof DomTextInput) {
```

Comment:
I don't think this should reside in the `PARENT ELEMENTS` block. Should be moved back, down to the `NON-PARENT ELEMENTS` block.